### DISCUSSION TITLE
Some enhancement suggestions

### DISCUSSION THREAD

**Discussion by newbeelearn** [mearashadowfax/ScrewFast#33] [Ideas]
First of all congrats and thank you for making such a beautiful theme. I am quite critical of everything and rarely praise but this is really good theme both in terms of looks and basic features(SEO) that are generally ignored by other themes.

Please list this theme in astro website, i accidentally discovered it.

 Here are some enhancement suggestions to make it perfect for small business
 - Add support for internationalization i18n is already supported in latest astro. This is not only for enabling this theme for people wanting to use it in other languages but also because adding internationalization at a later stage becomes difficult if support is not there.
 - Add support to share blog articles on social media. This helps readers to share the content on other channels.
 - Add support for toc on the side in blog article. For long articles navigating becomes difficult if toc is not present.
 - Add support for docs. Docs and blog articles have slightly different navigation and features(search) and almost every saas requires  docs page. Integrating it to this theme would make it one stop solution for all the things home/blog/docs.

**Comment by mearashadowfax** [comment]
@newbeelearn, thank you for your feedback and suggestions for enhancing the theme!

Your suggestions are fantastic and will surely contribute to making the theme even better. I'll definitely consider them and look into updating the template accordingly.

Regarding the listing, the template was recently added to the  [Astro Themes website](https://astro.build/themes/details/screwfast/).

Thanks again for your input!

**Comment by zicklag** [comment]
I just found this theme by chance a couple days ago, and I agree that it is quite impressive. Great work and thanks for sharing! 🌟

**Comment by mearashadowfax** [comment]
I thought it'd be good to summarize the progress after a week:

- Some new components were added to the theme: social share button, bookmark functionality, post feedback block, and a neat icon component.
- Starlight now serves as a docs theme, complete with i18n support and some other tweaks.
- Big thanks to @newbeelearn for the PRs - we've had a solid refactoring session that made components way more reusable and tidied up the architecture a bit.
- On the roadmap: integrating i18n with the main site, adding toc for posts, and more refactoring.

**Comment by newbeelearn** [comment]
FYI, i wrote an article on why i chose this theme
[article](https://newbeelearn.com/blog/best-astro-theme-saas-2024)

I was thinking if we should put Who is using the theme somewhere in readme and link to sites using this theme. It will act as social proof.

**Comment by mearashadowfax** [comment]
Nice article! And yes, we can include it once we have enough sites to showcase.

**Comment by bribbon** [comment]
Thank you so much for this beautiful theme. The fact that i18n is supported in the docs is brilliant 👌
Is it possible to make the blog multilingual? So that the language changer also changes the language in a blog post?

I haven't found a way to set this up yet. What do you think, is this possible and if so, how?

**Reply by mearashadowfax** [reply]
Hello, @bribbon ! Sorry for the delayed response, I missed the notification. It should indeed be possible. I've managed to organize the blog with sub-directories and filter blog posts by language.  Please see the latest update.The implementation might not be the best design and may require manual adjustments to the `LanguagePicker` component based on the project structure, but it works.

**Reply by bribbon** [reply]
Hello @mearashadowfax, thank you for your reply.
I'm delighted that we can now all use i18n on the blog. Thank you very much for your contribution/repo!
Best wishes from Switzerland! bribbon
